### ISSUE TITLE
Fails when using a Google Doc URL sources

### ISSUE THREAD

**Issue by donert** [rse/vingester#16]
The browser window will show up, survive a few seconds and then close and the NDI feed ends.
Every time I “Start” I get a request to allow incoming connections. If I turn off the firewall, the message does not appear, but the NDI feed is still not stable.

Here is a test URL. 
https://docs.google.com/document/d/16aRNFAzKUoAYdkZ2zWMEPE6ud4wTicDNY1IuDHoMyfo/edit?usp=sharing

Vingester version 1.6.0 (it happened with the previous version as well.
MacOS 10.14.6

It works fine for www.google.com.

**Comment by donert** [comment]
I found a work around. I created a local html document that contains the link to the google doc. I can start the NDI feed and then after that is up and running, click on the link. It stays up. It still seems a little flaky. I have to try a few times. I did find this message in the console "refused to display 'https://accounts.google.com/' in a frame because it is set to 'X-frame-Options' to 'deny'"

**Comment by rse** [comment]
I was able to repeat your problem. I've still no clue what exactly is the problem why the browser closes, but it might be related to the security issue you traced. Let's see what I can find out...

**Comment by rse** [comment]
Yes, indeed. It is a security issue. The console also tells me:

```
19:09:54.285 > [info]   (main) browser: worker: started
19:09:54.291 > [info]   (main) browser/worker-7A16E4D3: setting capture framerate to 0/30/59 (initially)
19:09:55.550 > [error]  (main) browser/content-7A16E4D3: console: Refused to display 'https://accounts.google.com/' in a frame because it set 'X-Frame-Options' to 'deny'.
19:09:55.557 > [info]   (main) browser: content: failed
19:09:55.558 > [info]   (main) browser: stop
```

So, the underlying Chromium engine refuses the start the URL in the browser window. How this can be changed I have to figure out. Especially what this "X-Frame-Options" is about...

**Comment by donert** [comment]
I forgot to mention that this has worked before. Just not sure which version is was. I think I downloaded it about april 11th.

**Comment by rse** [comment]
This is now fixed in Vingester 1.6.1 by no longer stopping a browser instance just because an error is thrown by a sub-frame of the Web Contents. This is similar to what real browsers also do. Thanks for reporting.
